### ISSUE TITLE
Rename Contract.wallet to Contract.account

### ISSUE THREAD

**Issue by marcelomorgado** [tasitlabs/TasitSDK#374]
> Or maybe setSenderAddress and getSenderAddress in case we want to set it to a contract-based account?

Getter/Setter should be renamed too.
Refs: https://github.com/tasitlabs/TasitSDK/pull/357#discussion_r279087773

**Comment by marcelomorgado** [comment]
Good question. We can have these situations:
- Same account as the signer and sender;
- An account as sender and one as the signer;
- An account as sender and many as the signers;

**Comment by pcowgill** [comment]
> Good question. We can have these situations:
> 
>     * Same account as the signer and sender;
> 
>     * An account as sender and one as the signer;
> 
>     * An account as sender and many as the signers;

Yep. Supporting and testing all of those situations is important (and maybe outside the scope of this issue based on its current name?)

**Comment by marcelomorgado** [comment]
For short-term we can just rename from `wallet` to `account and then split this issue to continue the discussion of how should be the Contract API to support the situations above. Make sense?

**Comment by pcowgill** [comment]
> For short-term we can just rename from `wallet` to `account and then split this issue to continue the discussion of how should be the Contract API to support the situations above. Make sense?

Yep, perfect!

**Comment by marcelomorgado** [comment]
Discussion about API extracted to: https://github.com/tasitlabs/TasitSDK/issues/375
